packages/graphql/index.js

```diff
@@ -9,20 +9,21 @@ import {
 import GlobalID from '@kiwicom/graphql-global-id';
 
 import fetch from './src/services/Fetch';
+import AllFlights from './src/queries/AllFlights';
+import createContext from './src/services/GraphQLContext';
 
 const schema = new GraphQLSchema({
   query: new GraphQLObjectType({
     name: 'RootQueryType',
     fields: {
+      allFlights: AllFlights,
```

Comment:
I hope it's a right time to discuss some naming conventions. I suggest not to use prefix `all`, we stoped using it on graphql.kiwi.com it became something like "habit you can't explain"

Reply:
I agree with you prefix **all** is kind of confusing. And based naming conventions on kiwi's glossary make sense. I would like to share this naming convention as a part of our documentation.

Reply:
I agree, I will come up with a better name